src/cljs/main/broadfcui/page/billing/billing_management.cljs

```diff
@@ -189,11 +189,13 @@
    :billing
    {:component Page
     :regex #"billing"
+    :terra-redirect #(str "billing")
     :make-props (fn [_] {})
     :make-path (fn [] "billing")})
   (nav/defpath
    :billing-project
    {:component Page
     :regex #"billing/([^/]+)"
+    :terra-redirect (fn [args] (str "billing?type=project&selectedName=" (:project-name args)))
```

Comment:
So it doesn't appear to cause a problem (in that the url works either way) but note that the order is switched between `type=project` and `selectedName=` in this url compared to the one generated in Terra